server/package.json

```diff
@@ -12,6 +12,7 @@
   "license": "ISC",
   "dependencies": {
     "express": "^4.17.1",
+    "nodemon": "^2.0.20",
```

Comment:
And add `nodemon` as a dev dependency as it is only needed for development purposes